app/models/publisher.rb

```diff
@@ -5,9 +5,9 @@ class Publisher < ApplicationRecord
   UPHOLD_CODE_TIMEOUT = 5.minutes
   UPHOLD_ACCESS_PARAMS_TIMEOUT = 2.hours
   PROMO_STATS_UPDATE_DELAY = 10.minutes
-  ADMIN = "admin"
-  PARTNER = "partner"
-  PUBLISHER = "publisher"
+  ADMIN = "admin".freeze
```

Comment:
could be easier to add this to the top of the file

```ruby
# frozen_string_literal: true
```

Reply:
I'm hesitant to take this on unless I'm certain that all strings within the model are immutable.